playwright-e2e/tests/lms/lms-adult-enrollment.spec.ts

```diff
@@ -0,0 +1,237 @@
+/* eslint-disable max-len */
+import { expect, Page } from '@playwright/test';
+import * as auth from 'authentication/auth-lms';
+import * as user from 'data/fake-user.json';
+import AdditionalConsentPage from 'dss/pages/lms/additional-consent-page';
+import HomePage from 'dss/pages/lms/home-page';
+import GetStartedPage from 'dss/pages/lms/get-started-page';
+import ResearchConsentPage from 'dss/pages/lms/research-consent-page';
+import MedicalReleasePage from 'dss/pages/lms/medical-release-page';
+import SurveyAboutLmsPage from 'dss/pages/lms/survey-about-lms-page';
+import SurveyAboutYou from 'dss/pages/survey-about-you';
+import { test } from 'fixtures/lms-fixture';
+import { assertActivityHeader } from 'utils/assertion-helper';
+import { generateUserName } from 'utils/faker-utils';
+import { logParticipantCreated } from 'utils/log-utils';
+import { waitForResponse } from 'utils/test-utils';
+
+test.describe.serial('LMS Adult Enrollment', () => {
+  let researchConsentPage: ResearchConsentPage;
+  let additionalConsentPage: AdditionalConsentPage;
+
+  const assertActiveActivityStep = async (page: Page, expectedText: string) => {
+    await expect(page.locator('.activity-step.active')).toHaveText(expectedText);
+  };
+
+  test('self-consent @visual @enrollment @lms', async ({ page }) => {
+    researchConsentPage = new ResearchConsentPage(page);
+    additionalConsentPage = new AdditionalConsentPage(page);
+
+    const participant = user.adult;
+    const firstName = generateUserName(participant.firstName);
+    const lastName = generateUserName(participant.lastName);
+    const fullName = `${firstName} ${lastName}`;
+
+    await test.step('Create account with an email', async () => {
```

Comment:
interesting! Will make sure to use this later

Reply:
Agree!